packages/crypto/tests/wycheproof_secp256r1.rs

```diff
@@ -0,0 +1,394 @@
+#![allow(clippy::single_match)]
+
+use cosmwasm_crypto::secp256r1_verify;
+use serde::Deserialize;
+
+// See ./testdata/wycheproof/README.md for how to get/update those files
+const SECP256R1_SHA256: &str = "./testdata/wycheproof/ecdsa_secp256r1_sha256_test.json";
+const SECP256R1_SHA512: &str = "./testdata/wycheproof/ecdsa_secp256r1_sha512_test.json";
+const SECP256R1_SHA3_256: &str = "./testdata/wycheproof/ecdsa_secp256r1_sha3_256_test.json";
+const SECP256R1_SHA3_512: &str = "./testdata/wycheproof/ecdsa_secp256r1_sha3_512_test.json";
+
+#[derive(Deserialize, Debug)]
+#[serde(rename_all = "camelCase")]
```

Comment:
Why the horrible camelCase? Just curious.

Reply:
I just download the files from the project where they use camelCase in JSON. I'd rather copy the JSONs 1:1 than changing the keys. See README.md for the data source.